bot/exts/moderation/infraction/_scheduler.py

```diff
@@ -226,7 +223,13 @@ async def apply_infraction(
                 log.error(f"Deletion of {infr_type} infraction #{id_} failed with error code {e.status}.")
             infr_message = ""
         else:
+            log.info(f"Applied {purge}{infr_type} infraction #{id_} to {user}.")
             infr_message = f" **{purge}{' '.join(infr_type.split('_'))}** to {user.mention}{expiry_msg}{end_msg}"
+            if infr_type not in ("ban", "kick"):
```

Comment:
```suggestion
            if infr_type not in {"ban", "kick"}:  # We have already notified the user for these types
```

Reply:
Have committed the comment but leaving as a tuple since that's what's used elsewhere in the codebase.